warrior/Actions/SnmpActions/common_snmp_actions.py

```diff
@@ -129,6 +134,7 @@ def snmp_get(self, snmp_ver, system_name, mib_name=None,
         port = snmp_param_dic.get('snmp_port')
 
         output_dict = {}
+        __custom_mib_paths = None
```

Comment:
Why variables starting with __?

Reply:
There are specific meanings for having variables with underscores. Please check https://shahriar.svbtle.com/underscores-in-python. By convention, we don't want any underscores at the beginning and end of our variables. Hence please remove them.

Reply:
same is updated.